docs/guides/getting-started.md

```diff
@@ -85,6 +85,8 @@ This command will:
 
 With spot/preemptible instances (`spot >= 0`), auto-recovery logic and persistent storage will be used to relaunch interrupted tasks.
 
+-> **Note:** Uploading the `workdir` may take a long time, depending on the quantity and size of the files it contains.
```

Comment:
> quantity

You mean 1000 x 1kB files is significantly slower than 1 x 1MB?

Reply:
```suggestion
-> **Note:** A large `workdir` may take a long time to upload.
```

Reply:
@casperdcl, should we do something similar for `terraform destroy` or just delete the second callout?